pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -146,7 +147,7 @@ func (q *QueryReconciler) runShowRestorePoints(ctx context.Context, dispatcher v
 	opts []showrestorepoints.Option) (err error) {
 	// set Querying status condition ,state and restore points prior to calling vclusterops API
 	err = vrpqstatus.Update(ctx, q.VRec.Client, q.VRec.Log, q.Vrpq,
-		[]*metav1.Condition{vapi.MakeCondition(v1beta1.Querying, metav1.ConditionTrue, "Started")}, stateQuerying, nil)
```

Comment:
I'm guessing this change is needed because the restorePoints list is required now? Did you try setting the optional marker?
```
// +kubebuilder:validation:Optional
```

Reply:
Oops, why didn't I consider this setting? Yes, this setting is better. We don't need to update nil to vops.RestorePoints{}, and it still returns an empty status for Restore Points in the CR